src/Makefile.am

```diff
@@ -33,6 +33,10 @@ endif
 if ENABLE_NEURON
   sources += nccl_ofi_interface_neuron.c
 else
+  AM_CPPFLAGS += $(CUDA_CPPFLAGS)
```

Comment:
nit: there's no reason to have these in the ENABLE_NVIDIA path.  They'll just be empty on Neuron.